index.html

```diff
@@ -57,5 +57,202 @@ <h1>I’m Happi. Glad to see you!</h1>
         </section>
     </section>
 
+  <section class="works">
+            <section class="cards">
+                <img src="./figmaImages/snapshoots/Snapshoot Portfolio.svg" alt="Snapshoot">
+           
+            <div class="cards-description">
+                <h2 class="cards-titles">Tonic</h2>
+                <div class="subheading-lists">
+                <span class="canopy">CANOPY</span>
+                <img src="./figmaImages/icons/circle-icon.svg" alt="">
+                <span class="subheading-list-title">Back End Dev</span>
+                <img src="./figmaImages/icons/circle-icon.svg" alt="">
+                <span class="subheading-list-title">2015</span>
+                </div>
+                <section class="card-content">
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or 
+                        sign-ups required.
+                    </p>
+                </section>
+                <ul class="webdev-languages">
+                    <li> <a href="#html">HTML</a> </li>
+                    <li> <a href="#css">CSS</a> </li>
+                    <li> <a href="#javascript">Javascript</a></li>
+                </ul>
+                <button class="see-project">See Project</button>
+            </div>
+            </section>
```

Comment:
- [ ] The works section is one entire section and as such should only have one `section` tag, in the main container. Every other element inside this section should use other semantically correct tags such as `div`'s, `ul's, etc. 
Kindly change the tag holding the card to a div tag and the tag holding the card content to a more appropriate tag such as a `p` tag.

### Note that since this card is repeated 4 times refactor the codebase everywhere, this change applies 🙏

Reply:
Okey
Thanks